flycheck.el

```diff
@@ -4335,6 +4348,23 @@ universal prefix arg, and only the id with normal prefix arg."
       (seq-do #'kill-new (reverse messages))
       (message (string-join messages "\n")))))
 
+(defun flycheck-explain-error-at-point ()
+  "Display an explanation for the first error at point."
+  (interactive)
+  (let* ((errors-at-point (flycheck-overlay-errors-at (point)))
+         (first-error (car errors-at-point))
```

Comment:
Well that was one of my questions as well.  Currently I have no examples for rust where there are multiple explanations at point, but trying to get all the explanations might not make the code logic more complex.  We would need to find a way to present them all to the user though.